### PR TITLE
Bengali Translation

### PR DESCRIPTION
**Pull request by SandipPalit**
## Title

This pull request (PR) is made in reference to: #286

## Description

This PR introduces Legesher to Bengali language.

## Motivation and Context

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Testing

- [ ] My submission passes the tests?
  - If you did additional testing, please describe in detail how you tested your changes
- [ ] I added new tests for core changes, as applicable?

### CODE REVIEW

File: locale/bn.yml [hunk 1]
```diff
@@ -0,0 +1,1813 @@
...
+  sorted: "সাজানো"
+  staticmethod: "স্ট্যাটিক পদ্ধতি"
+  str: "স্ট্রিং"
+  sum: "যোগফল"
+  super: "সুপার"
+  tuple: "টুপল"
+  type: "প্রকার"
+  vars: "যার"
+  zip: "জিপ"
+  __import__: "__আমদানি__"
```
Comment (madiedgar) [inline locale/bn.yml:1357]: This is fantastic @SandipPalit ! Thank you so much for adding the initial translation 👍  

As software always iterates, we found a better way to mark translation reviews from Hacktoberfest feedback. I'm still working on documenting the new process, but would you mind making the following change?

- [ ] Add your @ username in a new line under each translated word.
- [ ] Provide the emoji that best captures your confidence around the translation (see key in [Translation Guide: Step 7](https://docs.legesher.io/legesher-translations/translation-guide))
- [ ] Add any additional commentary or other words you considered in your own review line

Here's an example:
``` yml
  zip: "জিপ"
    # ✅  @SandipPalit
  __import__: "__আমদানি__"
    # ❓  @SandipPalit what about "this" ? or "that"?
```

Let me know if you have any questions!

**Review by madiedgar — CHANGES_REQUESTED** [review]

File: locale/bn.yml (hunk 1 above)
Comment (SandipPalit) [reply locale/bn.yml]: Hi @madiedgar, I have implemented the following changes.

- [x] Add your @ username in a new line under each translated word.
- [x] Provide the emoji that best captures your confidence around the translation.
- [x] Add any additional commentary or other words you considered in your own review line.

Please let me know if I can do even more improvements.

**Comment by SandipPalit** [comment]
Hi @madiedgar, I have implemented the following changes.

- [x] Add your @ username in a new line under each translated word.
- [x] Provide the emoji that best captures your confidence around the translation.
- [x] Add any additional commentary or other words you considered in your own review line.

Please let me know if I can do even more improvements.

**Review by madiedgar — APPROVED** [review]
Perfect! Thank you so much @SandipPalit 👍

**Comment by madiedgar** [comment]
Thank you so much @SandipPalit for your contribution! I'm going to open a couple new issues to keep the momentum with the Bengali translation. Would love your help! 👍 
- PR: Add Bengali as a [languages in active development](https://github.com/legesher/legesher-translations#languages-in-active-development) on the master branch. (The list is in alphabetical order.)
- share with your friends who are native Bengali speakers 👍 I'll create a new issue for them to review the translations!

**Pull request merged**